pam_ussh.go

```diff
@@ -179,14 +179,14 @@ func authenticate(w io.Writer, uid int, username, ca string, principals map[stri
 		}
 
 		if len(principals) == 0 {
-			pamLog("Authentication succeeded for %s, cert %d", cert.ValidPrincipals[0], cert.Serial)
+			pamLog("Authentication succeeded for %s, cert %d", username, cert.Serial)
```

Comment:
I'd prefer for this to be an optional behavior. Are you interested in also adding something like `accept_username` or `match_username` to the options parsing code to support this?

Reply:
sorry, it must've been early in the morning and I was without my coffee. when I looked at this diff earlier I somehow something completely different that what you put.

I'd prefer just add username to the log rather than replace ValidPrincipals[0] entirely. would that work for you?